### DISCUSSION TITLE
4.6.0.2版本全屏播放卡顿

### DISCUSSION THREAD

**Discussion by Docence** [ywmoyue/biliuwp-lite#73] [General]
全屏观看视频时，经常出现卡顿，4.5版本就已经有这个问题

**Comment by ywmoyue** [comment]
我没遇到过，请开一个性能监控软件（Xbox game bar/任务管理器/Aida64）检查播放卡顿时是否某个性能指标占满了或快占满了（CPU/内存/显卡/硬盘/网络）
